src/content/language/collections.md

```diff
@@ -242,78 +250,180 @@ final constantMap = const {2: 'helium', 10: 'neon', 18: 'argon'};
 For more information about maps, refer to the Maps section of the
 [`dart:core` documentation](/libraries/dart-core#maps).
 
-## Operators
+[Using constructors]: /language/classes#using-constructors
+[`Map`]: {{site.dart-api}}/dart-core/Map-class.html
+[type inference]: /language/type-system#type-inference
+
+## Supplementals
 
-### Spread operators
+The following supplemental materials can be used with
+Dart's collection types.
 
-Dart supports the **spread operator** (`...`) and the
-**null-aware spread operator** (`...?`) in list, map, and set literals.
-Spread operators provide a concise way to insert multiple values into a collection.
+<a id="control-flow-operators"></a>
+<a id="spread-operators"></a>
 
-For example, you can use the spread operator (`...`) to insert
-all the values of a list into another list:
+### Spread operations
+
+With a spread operation, you can
+add values from one collection to another collection. Null
+values inside of a  collection can be passed in, but null
+collections produce an error. A spread operation has this
```

Comment:
I don't know if it's worth pointing out that the thing you spread can't be `null`. There are many places in the language where `null` is an error (like `1 + null` or `null.someMethod`) and we don't usually bother to call that out.

If your intent is to contrast it with null-aware spread, that makes sense.

Reply:
Yes, my intent is to contrast with null-aware spread. I'll add a link to it to make this more clear. If more is needed, let me know.